bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx66.html

```diff
@@ -0,0 +1,182 @@
+
+  {% block site_header %}{% endblock %}
+
+  {% block content %}
+
+  <main class="mzp-t-firefox">
+    <header class="c-page-header">
+      <div class="mzp-l-content c-page-header-inner">
+        {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '147', 'height': '55', 'class': 'c-page-header-logo-fx'}) }}
+        <h1 class="c-page-header-up-to-date">{{ _('Congrats! You’re using the latest version of Firefox.') }}</h1>
+        <img src="{{ static('img/logos/mozilla/wordmark-dark.svg') }}" alt="Mozilla" width="112" height="32" class="c-page-header-logo-moz">
+      </div>
+    </header>
+
+    <section class="c-sticky-signup fxa-form-cta">
+        <div class="mzp-l-content">
+          <div class="c-sticky-signup-container">
+            <div class="c-sticky-signup-content">
+              <h2 class="c-sticky-signup-title">{{ _('Get a Firefox Account and all the benefits it unlocks. ') }}</h2>
+              <button type="button" class="sticky-dismiss">Dismiss this prompt.</button>
+            </div>
+            {{ fxa_email_form(
+                entrypoint='whatsnew',
+                button_class='mzp-c-button',
+                utm_source='whatsnew',
+                utm_params={'campaign': 'fxa-embedded-form', 'content': 'whatsnew', 'medium': 'referral'})
+            }}
+          </div>
+        </div>
+      </section>
+
+      {% if show_newsletter %}
+      <section class="c-sticky-signup newsletter-form-cta">
+        <div class="mzp-l-content">
+          <div class="c-sticky-signup-container">
+            <div class="c-sticky-signup-content">
+              <h2 class="c-sticky-signup-title">{{ _('Get tips to live your best online life right in your inbox.') }}</h2>
+              <button type="button" class="sticky-dismiss">Dismiss this prompt.</button>
+            </div>
+            <div class="mzp-c-newsletter">
+              {{ email_newsletter_form(
+                protocol_component=True,
+                include_title=False,
+                spinner_color='#fff',
+                button_class='mzp-c-button')
+              }}
+            </div>
+          </div>
+        </div>
+      </section>
+      {% endif %}
+
+    {% set content_blocking_image='firefox/whatsnew_64/hero.png' if DIR == 'ltr' else 'firefox/whatsnew_64/hero-rtl.png' %}
+    {% call hero(
+     title=_('One log-in. Power and security everywhere.'),
+     class='mzp-has-image mzp-l-reverse t-wn64 show-fxa-default show-fxa-supported-signed-out',
+     image_url=content_blocking_image,
+     include_highres_image=true,
+     include_cta=True
+    ) %}
+    <p class="show-fxa-supported-signed-out show-fxa-default">
+      <a href="https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=mozilla.org-fx.wnp64&utm_content=%2Ffirefox%2FWNP%2F64%2F&utm_campaign=wnp64&utm_source=wnp-64" class="mzp-c-button mzp-t-download">{{_('Create a Firefox Account') }}</a><br>
```

Comment:
We should probably use the newly crreated `fxa_link_fragment` macro here? https://github.com/mozilla/bedrock/blob/master/bedrock/base/templates/macros.html#L304